examples/hurricanes/main.js

```diff
@@ -369,8 +369,8 @@ $(function () {
     }
   );
 
-  // Create a feature layer to draw on
-  layer = map.createLayer('feature');
+  // Create a feature layer to draw on.
+  layer = map.createLayer('feature', {features: ['line.multicolor']});
```

Comment:
I see now that you have added options at the time of registration. Interesting idea but now we have to make sure we are consistent about it and document it as well.